### PR TITLE
Feat/cli

### PR DESCRIPTION
**Pull request by hiromon0125**
# Features
* Failed to add Arduino-Cli integration due to incompatibility with RaspberryPi and Arduino-Cli
* Add clear direction on how to set up Arduino extension in VSCode
* Setup arduino.json
* Setup gitignore
* Add sample blink code
* Set up repository to follow basic community standard
   - Add Code of conduct
   - Add Contributing
   - Add Security Policy
   - Add Pull Request Template
   - Add Issue Template

# How to Test
* Follow the directions on the ReadMe file and make sure all of the steps are clearly written
* Check to see if there are any errors in the main .ino file
* Make sure the main .ino file passes the verification of the Arduino extension

merged with #18

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Feature
```
Comment (cvr8924) [inline .github/pull_request_template.md:1]: Not a critique but what is the purpose of this file. is the intention that a Pull Request file will be included in future PRs or is this just the template for PR descriptions in git. Similar question with Bug Reports. Those are currently managed through trello, is there an in git methodology you'd wanna use for them? i'm not opposed if their is im just unfamiliar

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+All community leaders are obligated to respect the privacy and security of the
+reporter of any incident.
+
+## Enforcement Guidelines
+
+Community leaders will follow these Community Impact Guidelines in determining
+the consequences for any action they deem in violation of this Code of Conduct:
+
+### 1. Correction
```
Comment (cvr8924) [inline CODE_OF_CONDUCT.md:74]: I mean realistically this codebase is administered under MDRC bylaws which would have its own reprimand process as an RIT club

File: README.md [hunk 3]
```diff
@@ -7,6 +7,8 @@ Quadruped Hybrid Pneumatic-Electric Robot. Utilizes McKibben Muscles for bio-inp
 ### Install dependencies
 * install [python](https://www.python.org/downloads/)
 * install [nodejs](https://nodejs.org/en/download/)
+* install [C/C++ extention](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools) for VSCode
```
Comment (cvr8924) [inline README.md:10]: thank you for adding this its always so annoying when readme dependancies aren't accurate and its v/ easy to forger

File: README.md [hunk 4]
```diff
@@ -24,3 +26,23 @@ catbot
...
+* and make sure the code passes verification
+
+### Possible Errors on Setup
+Include path error:
+* Try editing the include path in `.vscode/c_cpp_properties.json`
+* Click on `Quick Fix...` at the bottom of the error popup menu after hovering on the error
+* Click on `Add to "include path": "..."`
+* If the error occures with multiple include path options try adding them using the wildcard (`**`) operator
+* Goto `.vscode.c_cpp_properties.json` in the root directory of the repo
+* Example: `"includePath": ["${workspaceFolder}/**", "/Users/doofenshmirtz/Library/Arduino15/packages/arduino/hardware/mbed_nano/4.0.8/cores/arduino/mbed/**"]`
```
Comment (cvr8924) [inline README.md]: "doofenshmirtz"
 lol

File: cli/index.js [hunk 5]
```diff
@@ -1,18 +1,9 @@
 #! /usr/bin/env node
```
Comment (cvr8924) [inline cli/index.js:1]: may be worth notating if this code is not currently in use / not functional just for posterity's sake.

**Review by cvr8924 — APPROVED** [review]
Looks good to me. Only major thing I have is we may wanna document somewhere what is isn't functional / in progress. Approved regardless tho

File: .github/pull_request_template.md (hunk 1 above)
Comment (hiromon0125) [reply .github/pull_request_template.md:1]: The purpose of this file is for when the user is making a pr this template is suggested automatically and is given to the developers. It sets a basic template of what a good pr should have.

Comment (hiromon0125) [reply .github/pull_request_template.md:1]: The template has two main things that every pr should contain: features description and direction to test the pr's feature. This makes it easy to test pr and understand them.

File: cli/index.js (hunk 5 above)
Comment (hiromon0125) [reply cli/index.js:1]: index is being used and can be used. install.js however is not and will be deprecated in the future is no proof of concept is made

Comment (hiromon0125) [reply cli/index.js:1]: running `npm -g install` and running `cb start` will run the main python program.

**Comment by hiromon0125** [comment]
code of conduct, README file, and cli README file has been updated to current code.

File: README.md [hunk 6]
```diff
@@ -24,3 +26,26 @@ catbot
...
+### Setup Arduino Extension
+* Open `src/arduino/arduino.ino` in VSCode
+* Click on `Mac`, `Win32`, or `linux` on the bottom right corner of the window and change to `Arduino`
+* Click on `<Select a Board Type>` in the bottom right corner
+* Search `Arduino Nano`
+* Select `Arduino Nano RP2040 Connect`
+* Go to command pallet (`Ctrl+Shift+P`)
+* Search `Arduino: Rebuild IntelliSense Configuration`
+* Click on Verify button on the top right corner of the window with a checkmark icon
+* and make sure the code passes verification
```
Comment (thatnoobles) [inline README.md:39]: Is this outdated?

**Review by thatnoobles — CHANGES_REQUESTED** [review]
With the introduction of the CLI v2, it look like a lot of this becomes obsolete. Can we just merge that one instead?

**Pull request closed without merging**